gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -224,11 +224,12 @@ def add_arguments(self, parser):
               'structure as input files with a suffix added to them.'))
     parser.add_argument(
         '--' + AnnotationOptions._VEP_IMAGE_FLAG,
-        default="",
+        default='gcr.io/gcp-variant-annotation/vep_91',
         help=('The URI of the docker image for VEP.'))
     parser.add_argument(
         '--' + AnnotationOptions._VEP_CACHE_FLAG,
-        default="",
+        default=('gs://gcp-variant-annotation-vep_cache/'
```

Comment:
nit: any reason why `vep_cache` is an underscore instead of a dash like the other parts?

Reply:
Done.
In general for file names I prefer '_' over '-' (hence the cache name file below) and the reason I picked 'gcp-variant-annotation' was that it was the name of the project. But I think it is better to be consistent so I changed the bucket name. Also given the trouble that I once had with GATK and '_' in bucket name maybe for bucket names I should change my habit.